examples/lib/intermediateHtmlStepUi.js

```diff
@@ -50,12 +50,64 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
         </section>\
         </div>';
   }
+
+  function generatePreview(previewStepName, customValues, path) {
```

Comment:
Also, how redundant is this with the original code? Would it be possible to have them share significant code? Thanks!

Reply:
Sure @jywarren Will do that.